src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java

```diff
@@ -216,7 +224,12 @@ public void run() {
           }
         }
 
+        // Read the Muted RCAs value
+        readMutedRcasFromConf();
```

Comment:
We can read and update the muted RCAs from conf here at once ?

Reply:
Done. Merged the read and muting the rcas in analysis graph into single function.